src/components/TextLink/TextLink.tsx

```diff
@@ -0,0 +1,31 @@
+import { TextLinkVariant } from './types';
+
+import type { ITextLink } from './types';
+
+import './TextLink.scss';
+
+const textLinkClass = {
+  [TextLinkVariant.Body1]: 'body1',
```

Comment:
Why do you need body1 if it already defaults? And I wouldn't say I like naming. Please make it more understandable

Reply:
Ok, i admit, it could be body1, but in that case, comment about default is still active